.travis.yml

```diff
@@ -63,22 +71,22 @@ matrix:
       <<: *_android
 
 before_install:
-- rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm
-  && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm
-  install $TRAVIS_NODE_VERSION
-- node --version
-- if [[ "$PLATFORM" =~ android ]]; then gradle --version; fi
-- if [[ "$PLATFORM" =~ ios ]]; then npm install -g ios-deploy; fi
-- if [[ "$PLATFORM" =~ android ]]; then echo y | android update sdk -u --filter android-22,android-23,android-24,android-25,android-26,android-27;
-  fi
-- git clone https://github.com/apache/cordova-paramedic /tmp/paramedic && pushd /tmp/paramedic
-  && npm install && popd
-- npm install -g cordova
+  # `language: android` has no Node.js installed, therefore we need to install it manually
+  - if [[ "$PLATFORM" =~ android ]]; then nvm install $TRAVIS_NODE_VERSION; fi
+  - node --version
+  - if [[ "$PLATFORM" =~ ios ]]; then npm install -g ios-deploy; fi
+  - if [[ "$PLATFORM" =~ local ]]; then npm install -g ios-deploy; fi
+  - if [[ "$PLATFORM" =~ android ]]; then gradle --version; fi
+  
+  - npm install -g github:apache/cordova-paramedic
+  - PARAMEDIC_BUILDNAME=travis-plugin-inappbrowser-$TRAVIS_JOB_NUMBER
+
+  - npm install -g cordova
 
 install:
-- npm install
+  - npm install
 
 script:
 - npm test
-- node /tmp/paramedic/main.js --config pr/$PLATFORM --plugin $(pwd) --shouldUseSauce
-  --buildName travis-plugin-inappbrowser-$TRAVIS_JOB_NUMBER
+- if [[ "$PLATFORM" != local ]]; then cordova-paramedic --config pr/$PLATFORM --plugin $(pwd) --buildName $PARAMEDIC_BUILDNAME --shouldUseSauce; fi
+- if [[ "$PLATFORM" =~ local ]]; then cordova-paramedic --config pr/local/ios-10.0 --plugin $(pwd) --buildName $PARAMEDIC_BUILDNAME; fi
```

Comment:
That's interesting. I dislike the hardcoding of `ios-10.0` down here, but like that it doesn't reroute through package.json/npm script.

Reply:
Since the `--config` value is append at paramedic's `conf` directory path, I was thinking what if platform was changed to `local/ios-10`...

Then the line will remain as
```
- if [[ "$PLATFORM" =~ local ]]; then cordova-paramedic --config pr/$PLATFORM --plugin $(pwd) --buildName $PARAMEDIC_BUILDNAME; fi
```

Though `PLATFORM=local/ios-10`, `"$PLATFORM" =~ local` will still match.

Reply:
Sounds good!